src/styles/form.scss

```diff
@@ -0,0 +1,51 @@
+.form {
+  width: 40%;
+
+  @media (max-width: $displayLaptop) {
+    margin-top: 32px;
+    width: auto;
```

Comment:
width: 100%

Reply:
I think auto is better. But ok

Reply:
it's better to show that you want to have it for the full width